code/obj/sec_tape.dm

```diff
@@ -0,0 +1,85 @@
+/obj/sec_tape
+	anchored = 1
+	icon = 'icons/obj/sec_tape.dmi'
+	icon_state = "sec_tape_s"
+	name = "security cordon"
+	real_name = "security cordon"
+	desc = "A small cordon of security tape, used to keep assistants off crime scenes."
+	density = 1
+	flags = FPRINT | USEDELAY | ON_BORDER
+	object_flags = HAS_DIRECTIONAL_BLOCKING
+	dir = SOUTH
+
+	New()
+		..()
+		update_icon()
```

Comment:
```suggestion
		UpdateIcon()
```
`UpdateIcon` is the "public" wrapper for `update_icon`, it calls `update_icon` plus does some additional bookkeeping like sending a "hey I just updated icon" signal in case anything cares about that.